src/argilla/client/feedback/training/schemas.py

```diff
@@ -973,12 +982,95 @@ def _prepare_for_training_with_trl(
         return ds
 
 
+class TrainingTaskForChatCompletionFormat(BaseModel):
+    """
+    Union[Tuple[str, str, str, str], List[Tuple[str, str, str, str]]]
+    """
```

Comment:
As @gabrielmbmb suggested I think this is unintended

```suggestion
```

Reply:
This is intended too because VSCode does not show a docstring when wanting to see the Pydantic model.